app/assets/stylesheets/application/lessons.scss

```diff
@@ -48,7 +48,7 @@
   }
 }
 
-.section-topics, .section-lessons {
```

Comment:
Things got weird here because there used to be separate .scss files for topics and lessons but I combined them into this sheet in #132. Looks like that was undone in a merge. Can you make sure all the additions from https://github.com/EFForg/sec/pull/132/files#diff-f9e46027640256ff89d655b6145fef5f remain in this file?

Reply:
The .collapse/.uncollapse/.meta rules from #135 also got nixed.

https://github.com/EFForg/sec/blob/bc4efcbce6c44e077f673260312bd630b811b3aa/app/assets/stylesheets/application/lessons.scss#L194-L208

Reply:
Okay, will fix in my next push later today.